bindings/rust/s2n-tls-sys/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "s2n-tls-sys"
+version = "0.1.0"
+authors = ["Cameron Bytheway <bythewc@amazon.com>"]
+edition = "2018"
+links = "s2n-tls"
+license = "Apache-2.0"
+
+[features]
+default = []
+pq = []
+quic = []
+
+[dependencies]
+libc = "0.2"
+openssl-sys = { version = "0.9" }
```

Comment:
whats the reasoning behind using `openssl-sys` vs `openssl`? I seems `-sys` lags behind so does that make is a more conservative choice?

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#-sys-packages

Reply:
openssl-sys will be what libs2n needs to link against. It's just the declarations of the c APIs and is responsible for the build/link step. Openssl would be a rust library that uses those declarations.